shared/signing/evm/evm_transaction.go

```diff
@@ -168,12 +168,22 @@ func WithEstimatedGasTipCap() TxBuildOption {
 
 func WithEstimatedGasFeeCap() TxBuildOption {
 	return func(ctx context.Context, b TxBuilder, tx *types.DynamicFeeTx) error {
-		price, err := b.rpc.SuggestGasPrice(ctx)
```

Comment:
this doesn't always work for 1559 tx's, so I changed it to how the client estimates the gas fee cap by taking the gas tip cap + (base fee of the most recent block * 2)